easybuild/easyblocks/q/qscintilla.py

```diff
@@ -136,11 +150,13 @@ def sanity_check_step(self):
         }
         # also install Python bindings if Python is included as a dependency
         python = get_software_root('Python')
+
+        run_cmd("python -c 'import sys; print(sys.path)'")
         custom_commands = []
         if python:
-            custom_paths['dirs'] += [os.path.join(det_pylibdir(), 'PyQt4'), os.path.join('qsci', 'api', 'python'),
-                                     os.path.join('share', 'sip', 'PyQt4')]
-            custom_commands = ["python -c 'import PyQt4.Qsci'"]
+            custom_paths['dirs'] += [os.path.join(det_pylibdir(), self.pyqt_version), os.path.join('qsci', 'api', 'python'),
```

Comment:
Use one entry per line, and use `.extend` rather than `+=` while you're at it:

```python
custom_paths['dirs'].extend([
    os.path.join(det_pylibdir(), self.pyqt_version),
    os.path.join('qsci', 'api', 'python'),
    os.path.join('share', 'sip', self.pyqt_version),
])